src/test/java/spark/utils/GzipUtilsTest.java

```diff
@@ -0,0 +1,43 @@
+package spark.utils;
+
+import com.diffblue.deeptestutils.Reflector;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class GzipUtilsTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  /* testedClasses: GzipUtils */
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 91 branch to line 92
+   */
+
+  @Test
+  public void testInputNullOutputFalse()
+      throws NoSuchMethodException, IllegalAccessException, InvocationTargetException {
+
```

Comment:
:-1: Too much reflection (unless reflection has been created correctly - i.e. no public constructor, in which case: :+1: